t/test_contracts_1.csv

```diff
@@ -0,0 +1,101 @@
+purchase_time,currency_code,options_type,buy_price,payout_price,sell_price,start_time,underlying_symbol,sell_time
+1464566400,USD,CALL,10,100,100,1464566400,PZero_Uncorrelated,1464567240
```

Comment:
Is `PZero_Uncorrelated` the underlying symbol? Can you choose a more descriptive name?